example_test.go

```diff
@@ -0,0 +1,183 @@
+package bluto_test
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/alibaba-go/bluto/bluto"
+	"github.com/alibaba-go/bluto/commander"
+	"github.com/gomodule/redigo/redis"
+)
+
+func Example_commandChain() {
```

Comment:
I think this is an advanced example and it's not understandable at first sight.
It would be better if we have a simpler example before this.